crates/uv-build/src/lib.rs

```diff
@@ -453,9 +493,7 @@ impl SourceBuild {
                     let resolved_requirements = build_context
                         .resolve(&default_backend.requirements)
                         .await
-                        .map_err(|err| {
-                            Error::RequirementsInstall("setup.py build (resolve)", err)
-                        })?;
+                        .map_err(|err| Error::RequirementsInstall("", err))?;
```

Comment:
```suggestion
                        .map_err(|err| {
                            Error::RequirementsInstall("setup.py build (resolve)", err)
                        })?;
```

We want to keep that message

Reply:
Wow, thanks, thats sloppy of me.